pantheon-bundle/frontend/src/app/searchResults.tsx

```diff
@@ -136,9 +136,10 @@ class SearchResults extends Component<IProps, ISearchState> {
           perPageLimit={this.state.pageLimit}
           showDropdownOptions={this.state.showDropdownOptions}
           bottom={this.state.bottom}
+          className="results__pagination"
         />}
 
-        {this.state.isEmptyResults && <EmptyState variant={EmptyStateVariant.small}>
+        {this.state.isEmptyResults && <EmptyState variant={EmptyStateVariant.small} className="results--empty">
```

Comment:
This is good except we previously used `search-results__section`, so to be consistent...

If this is equivalent to `<DrawerContentBody>` in it's role and styling (aka it's basically the same component, but different) I'd use:
`search-results--empty`

If it's wrapped by `<DrawerContentBody>` or an equivalent wrapper that's basically the same component I'd make the class:
`search-results__empty-wrapper`

Since it's a part of `search-results`.